etc/latex-common.profile

```diff
@@ -0,0 +1,32 @@
+# Firejail profile for latex-common
+# This file is overwritten after every install/update
+# Persistent local customizations
+include latex-common.local
+# Persistent global definitions
+# added by caller profile
+#include globals.local
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+
```

Comment:
Can we add `include whitelist-var-common.inc`?
IIRC latex additional requires `whitelist /var/lib` and maybe `writable-var`

Reply:
Did not think of that. I had no issues building my documents.
Why would you make it writable?

As far as I can see it uses this directory for *custom* latex packages, user defined packages should live in *~/texmf*.

Reply:
> Why would you make it writable?

Just remembering some latex related issues i've got in the past. However, can we add it or not? Adding it would be more restrictive.